### PR TITLE
Update all references from sourcecode.cio.gov to the whitehouse.gov pdf

### PR DESCRIPTION
**Pull request by Sgtpluck**
This PR fixes/implements the following **bugs/features**
 
* updates the README directions for starting this project (the node version was outdated)
* updates all references for the FCSP sourcecode.cio.gov links to the whitehouse.gov pdf 
* fixes a broken test snapshot
* updates software.cio.gov link references to the appropriate legacy pdf

Explain the **motivation** for making this change. What existing problem does the pull request solve?

This PR fixes broken links

### CODE REVIEW

**Comment by Sgtpluck** [comment]
hmm @drewbo i was able to get the tests to fail locally when i updated the snapshots initially, but i can't get it to fail in the same way that it is now locally, and i don't really understand what the tests are doing. do you have any thoughts before i spend time diving into it?

File: README.md [hunk 1]
```diff
@@ -27,9 +27,9 @@ Here’s how you can help contribute to code.gov:
 
 ## Getting Started
 
-You will need node to run this website. It's built against v10.19.0. The best way to get node is to install it via `nvm`. See the [nvm installation instructions](https://github.com/nvm-sh/nvm/blob/master/README.md#installation-and-update) to set it up on your system.
+You will need node to run this website. It's built against v14.21.2 (`lts/fermium`). The best way to get node is to install it via `nvm`. See the [nvm installation instructions](https://github.com/nvm-sh/nvm/blob/master/README.md#installation-and-update) to set it up on your system. 
 
-After you have cloned this repo, you can use `npm install` to install all of the
+A fter you have cloned this repo, you can use `npm install` to install all of the
```
Comment (drewbo) [inline README.md:32]:
```suggestion
After you have cloned this repo, you can use `npm install` to install all of the
```

**Review by drewbo — APPROVED** [review]
Looks good, there was one random space added in the docs but otherwise good to go

**Comment by Sgtpluck** [comment]
@peterrowland are you the merger of this repo? neither drew nor i can, so tagging you in!

**Comment by peterrowland** [comment]
√

**Pull request merged**